boards/arm/b_l072z_lrwan1/b_072z_lrwan1.dts

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright (c) 2018 Aleksandr Makarov <aleksandr.makarov@mailbox.org>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/dts-v1/;
+#include <st/stm32l072.dtsi>
+
+/ {
+	model = "STMicroelectronics B-L072Z-LRWAN1 Discovery kit";
+	compatible = "st,stm32l072z-lrwan1", "st,stm32l072";
+
+	chosen {
+		zephyr,console = &usart2;
+		zephyr,sram = &sram0;
+		zephyr,flash = &flash0;
+	};
+};
```

Comment:
Please add support for user button (B1) and leds(LD1>LD4).

Reply:
Added support for those